### PR TITLE
feat: validation for :disable-commands config

### PR DESCRIPTION
**Pull request by opqdonut**
for #1858, follow-up to #1900

# Definition of Done / Review checklist

## Reviewability
- [x] link to issue

### CODE REVIEW

**Comment by opqdonut** [comment]
The warning looks like this:

```
2020-01-14 10:50:12,170 [nRepl-session-cce0e18d-472a-46ea-8217-c5f22b37296c - - -] WARN  rems.config - Unrecognized values in :disable-commands :  (:foobar)
2020-01-14 10:50:12,170 [nRepl-session-cce0e18d-472a-46ea-8217-c5f22b37296c - - -] WARN  rems.config - Supported-values:  (:application.command/copy-as-new :application.command/invite-member :application.command/submit :application.command/request-review :application.command/review :application.command/reject :application.command/add-licenses :application.command/remove-member :application.command/revoke :application.command/request-decision :application.command/accept-licenses :application.command/accept-invitation :application.command/decide :application.command/uninvite-member :application.command/remark :application.command/add-member :application.command/save-draft :application.command/approve :application.command/return :application.command/assign-external-id :application.command/close :application.command/change-resources :application.command/create)

File: src/clj/rems/config.clj [hunk 1]
```diff
@@ -27,13 +29,17 @@
     config))
 
 (defn- parse-config [config]
-  (update config :email-retry-period #(Period/parse %)))
+  (update config :email-retry-period #(Period/parse %))
```
Comment (okahilak) [inline src/clj/rems/config.clj]: Hmm, aren't the changes done on this line lost after the next line is evaluated?

Comment (opqdonut) [reply src/clj/rems/config.clj]: lol, amateur mistake, thanks!

Comment (okahilak) [reply src/clj/rems/config.clj]: :)

File: src/clj/rems/config.clj [hunk 2]
```diff
@@ -27,13 +30,27 @@
     config))
 
 (defn- parse-config [config]
-  (update config :email-retry-period #(Period/parse %)))
+  (-> config
+      (update :email-retry-period #(Period/parse %))
+      (update :disable-commands (partial mapv keyword))))
+
+(deftest test-parse-config
```
Comment (okahilak) [inline src/clj/rems/config.clj]: Thanks for adding the test!

**Pull request merged**